### ISSUE TITLE
Feature request: ARM binaries or cross-compiling instructions

### ISSUE THREAD

**Issue by m4rkw** [duosecurity/duo_unix#201]
Hi

I'd like to deploy duo_unix on a busybox target that has no compiler toolchain or means of installing one. Would be awesome if you could provide either binary releases for different architectures or cross-compiling instructions.

I can probably figure out how to cross-compile it, if I get some time to figure it out I'll post the results here.

Cheers,
Mark

**Comment by m4rkw** [comment]
Got it working: https://github.com/duosecurity/duo_unix/pull/203

**Comment by vwbusguy** [comment]
We had to compile from source, but it also means we have different deployment methods for ARM servers than x86 for Duo.  Given how simple this is to compile, it seems like it shouldn't be much effort to just include the arm64/aarch64 binaries in the repos, too.

**Comment by aboschke** [comment]
Ping!
It's been four years, Cisco.  Roll these ARM packages.

**Comment by mkato** [comment]
Yes please.  We have shifted our entire Linux AWS fleet to aarch64 over the last few years, and now could really use arm packages for Ubuntu.  I believe many enterprise customers on AWS and Azure utilize considerable amount of arm instances.
